src/main/java/net/javadiscord/javabot/systems/user_commands/leaderboard/QOTWLeaderboardSubcommand.java

```diff
@@ -160,7 +161,7 @@ private void drawUserCard(@NotNull Graphics2D g2d, @NotNull Member member, QOTWP
 
 			boolean left = true;
 			int y = logo.getHeight() + 3 * MARGIN;
-			for (Member m : topMembers) {
+			for (Member m : topMembers.stream().map(Pair::second).toList()) {
```

Comment:
Why are you using a foreach loop and streams together?

Reply:
Cause otherwise I would have to use AtomicReferences. I'll clean up this command later

Reply:
Couldn't you just do something like that?
```java
for (Pair<QOTWAccount, Member> memberPair : topMembers) {
	Member m = memberPair.second();
}
```

Reply:
well yeah